Cargo.toml

```diff
@@ -245,6 +245,7 @@ level = 'warn'
 check-cfg = [
     'cfg(mariadb, values(any()))',
     'cfg(sqlite_ipaddr)',
+    'cfg(postgres, values(any()))',
```

Comment:
This is kinda dumb but could you move this up one line? It bugs me that it splits two `sqlite` cfg's.

Reply:
Strange, I usually alphabetize things like this

Reply:
Reordered